fec.config.js

```diff
@@ -1,5 +1,5 @@
 module.exports = {
-  appUrl: ['/user-preferences/notifications'],
+  appUrl: ['/my-notification-preferences/notifications'],
```

Comment:
Let's not change the URL as that would require additional work on Akamai redirects.